examples/compare_connectivity_over_time_over_trial.py

```diff
@@ -0,0 +1,401 @@
+# time for single-trials or from experiments that do not involve exactly
+# repeated trials, then :func:`mne_connectivity.spectral_connectivity_time`
+# should be employed. This function also takes data in the form of
+# :class:`mne.Epochs`, but it may consist of a single epoch.
+# If there are multiple epochs, the connectivity over time is computed for
+# epoch separately, with the option to average over epochs.
+#
+# To better illustrate the differences between the two functions,
+# we will employ them on two simulated cases.
+
+# Author: Qianliang Li <glia@dtu.dk>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import matplotlib.pyplot as plt
+import mne
+
+from mne_connectivity import (spectral_connectivity_epochs,
+                              spectral_connectivity_time)
+from mne.datasets import sample
+
+rng = np.random.default_rng(1234)  # set seed for reproducibility
+
+print(__doc__)
+
+###############################################################################
+# Simulated examples
+# ------------------
+# **Case 1: Repetitions of the same trial.**
+#
+# Let's generate some simulated data in the format of :class:`mne.EpochsArray`.
+# In this case, we will use random data for 3 channels over 5 epochs, but
+# all the epochs are just exact replicates of the first epoch. This simulates
+# when data is collected over an event of interest where we **assume** the
+# connectivity structure is the same over each event.
+
+n_epochs = 5  # number of simulated epochs
+n_channels = 3  # number of channels
+n_times = 2000  # number of sample points
+sfreq = 250  # Set sampling freq
+data = rng.random((n_epochs, n_channels, n_times))  # generate random data
+
+# In case 1, we overwrite all epochs with the data from the first epoch
+for i in range(n_epochs):
+    data[i] = data[0]
+
+ch_names = ["C3", "Cz", "C4"]  # three random channel names
+info = mne.create_info(ch_names, sfreq, ch_types="eeg")  # create info object
+data_epoch = mne.EpochsArray(data, info)  # create EpochsArray
+
+data_epoch.plot(scalings=0.75)  # Visualize the data
+
+###############################################################################
+# First we compute connectivity over trials.
+
+# Freq bands of interest
+Freq_Bands = {"theta": [4.0, 8.0],
+              "alpha": [8.0, 13.0],
+              "beta": [13.0, 30.0]}
+n_freq_bands = len(Freq_Bands)
+min_freq = np.min(list(Freq_Bands.values()))
+max_freq = np.max(list(Freq_Bands.values()))
+
+# Provide the freq points
+freqs = np.linspace(min_freq, max_freq, int((max_freq - min_freq) * 4 + 1))
+
+# The dictionary with frequencies are converted to tuples for the function
+fmin = tuple([list(Freq_Bands.values())[f][0] for f in range(len(Freq_Bands))])
+fmax = tuple([list(Freq_Bands.values())[f][1] for f in range(len(Freq_Bands))])
+
+# We specify the connectivity measurements
+connectivity_methods = ["plv", "wpli"]
+n_con_methods = len(connectivity_methods)
```

Comment:
The choice was actually because I implemented my own plv and wpli functions over time a few years ago, as MNE-connectivity didn't have the functionality, and wanted to test it up against it (Issue #17)

But I have now changed it to use two more different measurements (coh and plv) as that might make more sense.